Rubberduck.VBEEditor/SafeComWrappers/SafeRedirectedEventedComWrapper.cs

```diff
@@ -23,6 +23,7 @@ protected SafeRedirectedEventedComWrapper(TSource target, TEventSource eventSour
         protected override void Dispose(bool disposing)
         {
             DetachEvents();
+            Marshal.ReleaseComObject(_eventSource);
```

Comment:
I don't think that's the right place. It should be inside the `DetachEvents`. Otherwise, you end up leaking when multiple `AttachEvents`/`DetachEvents` calls are made to the evented wrapper. The `Dispose` will call the `DetachEvents` so there should be no leak that way.